### PR TITLE
Add all variants of clamp op

### PR DESCRIPTION
**Pull request by xytintel**
e.g. clamp.Tensor_out, clamp_min.Tensor_out, clamp_max.Tensor_out

### CODE REVIEW

**Comment by xytintel** [comment]
image
image
image

File: yaml/xpu_functions.yaml [hunk 1]
```diff
@@ -185,6 +185,9 @@ supported:
   - clamp.out
   - clamp_min.out
   - clamp_max.out
+  - clamp.Tensor_out
+  - clamp_min.Tensor_out
```
Comment (fengyuan14) [inline yaml/xpu_functions.yaml:189]: We need add all variants.
image

File: src/aten/TensorCompare.cpp [hunk 2]
```diff
@@ -248,6 +267,77 @@ Tensor& XPUNativeFunctions::clamp_max_out(
...
+Tensor& XPUNativeFunctions::clamp_max_out(
+    const Tensor& self,
+    const Tensor& max,
+    Tensor& result) {
+  auto iter = clamp_max_tensor_meta(self, max, result);
+  native::xpu::minimum_kernel(iter);
+  return result;
+}
+
+TensorIterator clamp_tensor_meta(
```
Comment (fengyuan14) [inline src/aten/TensorCompare.cpp:288]: clamp_meta ?

Comment (fengyuan14) [reply src/aten/TensorCompare.cpp:288]: image

**Comment by fengyuan14** [comment]
Have got fine-gran cases covered.
image

File: yaml/xpu_functions.yaml (hunk 1 above)
Comment (xytintel) [reply yaml/xpu_functions.yaml]: done

**Review by fengyuan14 — APPROVED** [review]

**Comment by fengyuan14** [comment]
clamp, clamp_max, clamp_min, have been added to fine grain cases in the previous commit.

**Pull request merged**